posthog/settings.py

```diff
@@ -731,3 +732,10 @@ def show_toolbar(request):
 
 # keep in sync with plugin-server
 EVENTS_DEAD_LETTER_QUEUE_STATSD_METRIC = "events_added_to_dead_letter_queue"
+
+VERSION_REQUIREMENTS = [VersionRequirement(service="postgresql", supported_version=">=12.0.0,<15.0.0",)]
```

Comment:
I think we need 11 to 13 here or we'll break cloud (see [this](https://github.com/PostHog/posthog/issues/6624))

Reply:
Yeah just checked this